src/backends/beamcoder.ts

```diff
@@ -152,6 +140,13 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
      */
     #streamIndex = 0;
 
+    /**
+     * Sets the rate at which the media is being played back. This is used to implement user controls for fast-forward,
```

Comment:
> user controls for fast-forward

I don't understand this part

Reply:
User as in API user?

Reply:
I copied it from here https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playbackRate